spec/integration/apply_compile_spec.rb

```diff
@@ -234,7 +234,7 @@ def get_notifies(result)
           expect(notify[0]['title']).to eq("Num Targets: 3")
           expect(notify[1]['title']).to eq("Target 1 Facts: {operatingsystem => Ubuntu, added => fact}")
           expect(notify[2]['title']).to eq("Target 1 Vars: {environment => production, features => [puppet-agent]}")
-          res = "Target 0 Config: Target('foo', {\"connect-timeout\"=>11, \"tty\"=>false, \"host-key-check\"=>false})"
+          res = "Target 0 Config: Target('foo', {})"
```

Comment:
The purpose of this particular test is to prove that inventory options are respected when querying inventory in an apply block. Will need to modify the plan to return the options instead of implicitly calling `to_s` with `#{}`.

Reply:
Are there other opts we should include in the list? Before making the changes here.

Reply:
Would it be a pain to have it configurable? Make optional arg with default value of empty hash. The default (empty hash) could just return the `uri` then let users supply a hash of options. Maybe a special case for "all" in the case someone wants to quickly debug something?

Reply:
Probably not worth making configurable. In a plan you can just access the `options` attribute.
`notify { "Target 0 Config: ${t[0].options}": }`